dotcom-rendering/src/components/FrontPagination.tsx

```diff
@@ -167,7 +166,16 @@ export const FrontPagination = ({
 								</a>
 							)}
 							{shouldSuffixDots && (
-								<span css={paginationDotsCss}>...</span>
+								<span
+									css={[
+										paginationDotsCss,
+										css`
+											margin-right: ${space[1]}px;
+										`,
+									]}
+								>
+									...
+								</span>
```

Comment:
If you wish to follow Emotion best practice, you [should use the style attribute for dynamic styles](https://emotion.sh/docs/best-practices#use-the-style-prop-for-dynamic-styles).